### ISSUE TITLE
Feature: Allow for prompting for passwords?

### ISSUE THREAD

**Issue by nixwiz** [mylesw42/osc#5]
While I certainly appreciate the quick switching this provides, how about allowing for prompting for password(s) should someone not like the idea of storing them?

```
localhost:
  env: dev
  username: admin
  password: [prompt]
  api: http://localhost:8080
  namespace: default
  format: tabular
  timeout: 15s
```

```
osc connect localhost
Password for admin: ********
Connected to Sensu backend: localhost (http://localhost:8080
```

**Comment by mylesw42** [comment]
Good idea. I like having something like a flag to enable the feature in the config profile. Cause maybe someone has a password of "[prompt]" ;)

```
localhost:
  env: dev
  interactive: true
  username: admin
  password: ""
  api: http://localhost:8080
  namespace: default
  format: tabular
  timeout: 15s
```

If interactive is true, then we could load the prompt for password input.

**Comment by nixwiz** [comment]
I like that idea.

**Comment by mylesw42** [comment]
This is now implemented in v0.3.0

**Comment by nixwiz** [comment]
Works like a champ!  Thanks!
